WordPress/Classes/System/WordPressAppDelegate.m

```diff
@@ -540,17 +542,25 @@ - (BOOL)isWelcomeScreenVisible
     return [presentedViewController.visibleViewController isKindOfClass:[LoginViewController class]];
 }
 
-- (BOOL)noBlogsAndNoWordPressDotComAccount
+- (BOOL)noWordPressDotComAccount
 {
     NSManagedObjectContext *context = [[ContextManager sharedInstance] mainContext];
     AccountService *accountService = [[AccountService alloc] initWithManagedObjectContext:context];
-    BlogService *blogService = [[BlogService alloc] initWithManagedObjectContext:context];
     WPAccount *defaultAccount = [accountService defaultWordPressComAccount];
 
+    return !defaultAccount;
+}
+
+- (BOOL)noBlogs
```

Comment:
(Sorry, i know this is actually a split of the previous method), but, could it be possible to rename this helper, perhaps, as `noSelfHostedBlogs`?.

At glance, you might think that this returns *true* if there are no blogs set up. However, you might have a dotcom account with multiple blogs, and this method would still return false.

Thanks!